src/board/board.ts

```diff
@@ -0,0 +1,37 @@
+import chalk from "chalk";
+import {drawChalkText, tileDataInterface} from "../tile/tile";
```

Comment:
interface names should start with a capital letter. Besides, we don't need to put the type in the name, so it should be `TileData`.

Reply:
Done